plugin.rb

```diff
@@ -167,28 +167,24 @@ def post_process(post)
     Guardian.class_eval { prepend DiscourseTranslator::GuardianExtension }
   end
 
-  class ::PostSerializer
-    attributes :can_translate
-
-    def can_translate
-      if !(
-           SiteSetting.translator_enabled && scope.user_group_allowed? &&
-             scope.post_group_allowed?(object)
-         )
-        return false
-      end
+  add_to_serializer :post, :can_translate do
```

Comment:
You might want to add a `include_condition` here too to just check `SiteSetting.translator_enabled`

Reply:
From /t/92657 if we don't include `define_include_method=false` we won't need to check if the plugin is enabled.

Reply:
Ahhh okay cool, good to know :+1: